conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptions.java

```diff
@@ -183,6 +188,13 @@ private static void frameworkException(HttpServerExchange exchange, FrameworkExc
         writeResponse(exchange, Optional.of(ConjureError.fromServiceException(exception)), statusCode);
     }
 
+    private static void deadlineExpiredException(HttpServerExchange exchange, DeadlineExpiredException exception) {
+        log.info("deadline exceeded", exception);
```

Comment:
should we specifically log the internal variant at `error` and everything else at `info`?

Reply:
sure, done